documentation/FortranIR.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-After semantic analysis is complete and it has been determined that the compiler has a legal Fortran program as input, the parse tree will be lowered to an intermediate representation for the purposes of high-level analysis and optimization.  In this document, that intermediate representation will be called Fortran IR or FIR.
+After semantic analysis is complete and it has been determined that the compiler has a legal Fortran program as input, the parse tree will be lowered to an intermediate representation for the purposes of high-level analysis and optimization.  In this document, that intermediate representation will be called Fortran IR or FIR.  The pass that converts from the parse tree and other data structures of the front-end to FIR will be called the "Burnside bridge".
```

Comment:
Is there a chance we could give this a name that is more descriptive of what the bridge does, like FrontendFIRBridge or similar? As per this section of the LLVM Style Guide: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Possibly at some point. For now and until the full scope and responsibilities are better defined, we'd prefer to use code names to refer to different passes.